packages/documentsearch/test/searchmodel.spec.ts

```diff
@@ -61,6 +62,19 @@ describe('documentsearch/searchmodel', () => {
       });
     });
 
+    describe('#parsingError', () => {
+      it('should set informative string message on invalid regex', async () => {
+        model.useRegex = true;
+        expect(model.parsingError).toEqual('');
+        const waitForStateChanged = signalToPromise(model.stateChanged);
+        model.searchExpression = 'query\\';
+        await waitForStateChanged;
```

Comment:
The failure in tests started appearing after 923de1a106ec8b9d93eac09c94f1fb177142e2df. This is because `stateChanged` gets emitted twice. Reverting to earlier version.

```suggestion
        model.searchExpression = 'query\\';
        await signalToPromise(model.stateChanged);
```